### PR TITLE
fix alibi logits bias order

### PR DESCRIPTION
**Pull request by gyin94**
In `apply_attention_logit_biases` function, attention_logit_biases would be cased to logits dtype. So the order matters to pass logits as the first argument
```python
return logits + attention_logit_biases.to(dtype=logits.dtype, device=logits.device)
```

### CODE REVIEW

File: axlearn/common/adapter_torch.py [hunk 1]
```diff
@@ -1387,7 +1387,7 @@ def forward(
         )
         # Causal mask.
         return apply_attention_logit_biases(
```
Comment (ruomingp) [inline axlearn/common/adapter_torch.py:1389]: Great catch! Could you change `apply_attention_logit_biases` to take `apply_attention_logit_biases` as a keyword arg? This also better matches https://docs.google.com/document/d/1tK3MyfZQgXyrvWNg3rt6NQCItuFnFw_FBd6-C7pE2Wk/edit#heading=h.d1xks5sf41jd.

Feel free to do so in a follow-up PR.

**Review by ruomingp — APPROVED** [review]
Is there a unittest that can catch this bug?

File: axlearn/common/adapter_torch.py (hunk 1 above)
Comment (gyin94) [reply axlearn/common/adapter_torch.py:1389]: sure. added a commit for this. please help review again. thanks.

Comment (ruomingp) [reply axlearn/common/adapter_torch.py:1389]: Thanks! Is there any other call site of `apply_attention_logit_biases`?

**Review by ruomingp — APPROVED** [review]

**Comment by ruomingp** [comment]
> Is there a unittest that can catch this bug?

Is it possible to add a unittest?

**Comment by gyin94** [comment]
@ruomingp I think so. I will add a follow up PR on this
